### ISSUE TITLE
Investigate and Add Single Sign-on Internal (SSOi) Domain to Socks

### ISSUE THREAD

**Issue by olivereri** [department-of-veterans-affairs/va.gov-cms#13773]
## Description
When testing PIV authentication it is a requirement to be on the VA Network via Citrix Access Gateway (CAG) or Government Furnished Equipment (GFE).  This is because, in part, SOCKS does not forward to IAM endpoint domains. It may be possible to add those domains to SOCKS however, there may be other connections required to transmit PIV credentials.

## Relations
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/13637

## Acceptance Criteria
- [ ] SSOi Domains are resolvable over SOCKS.
- [ ] CMS login via PIV Authentication works over SOCKS.

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by EWashb** [comment]
I'm going to throw this into #14595, although I feel like not entirely related (maybe I'm wrong). The mini orange module support option could potentially help us here. also, since CAG isn't a thing anymore...what happens next?

**Comment by BerniXiongA6** [comment]
Can this be closed? @EWashb @ndouglas

**Comment by ndouglas** [comment]
@BerniXiongA6 No, I think this is worth keeping open. Will defer to @olivereri though.

**Comment by EWashb** [comment]
I agree with that @ndouglas. We can remove it from being tied to Mini Orange since I don't think we want to go that route. This could be its own epic of work if we want to investigate alternate PIV auth.
